packages/babel-make-styles/__fixtures__/object/code.ts

```diff
@@ -0,0 +1,6 @@
+import { makeStyles } from '@fluentui/react-make-styles';
+
+export const useStyles = makeStyles({
+  root: { color: 'red', padding: '4px' },
+  icon: { background: 'green', marginLeft: '4px' },
```

Comment:
should this syntax be tested in a fixture ? (I don't know what it is called)

```suggestion
  [icon]: { background: 'green', marginLeft: '4px' },
```

Reply:
Good call, actually it works out of the box as we don't touch keys. I added a fixture anyway (4298f8d)